src/Behat/Testwork/EventDispatcher/TestworkEventDispatcher.php

```diff
@@ -17,28 +17,37 @@
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  */
-$isSymfony5AndPhpGreater72 = function () {
+$isSymfony5EventDispatcher = function () {
     $reflection  = new \ReflectionClass(\Symfony\Component\EventDispatcher\EventDispatcher::class);
     $dispatch    = $reflection->getMethod('dispatch');
+
+    if ($dispatch->getNumberOfParameters() === 1) {
+        // This is the 4.3 / 4.4 version, which has `public function dispatch($event/*, string $eventName = null*/)` and
```

Comment:
Great that you've written these comments 👍

Reply:
Always nice to know what on earth I was thinking later on :)